internal/server/iigointernal/legislature_test.go

```diff
@@ -166,36 +166,36 @@ func generateRulesTestStores() (map[string]rules.RuleMatrix, map[string]rules.Ru
 //		t.Errorf("The rules in play are not the same as expected, expected '%v', got '%v'", expectedStateTransfer, stateTransfer)
 //	}
 //}
-
-func (s *baseSpeaker) announceVotingResultTest() error {
-
-	var rule string
-	var result bool
-	var err error
-
-	if s.clientSpeaker != nil {
-		//Power to change what is declared completely, return "", _ for no announcement to occur
-		rule, result, err = s.clientSpeaker.DecideAnnouncement(s.RuleToVote, s.VotingResult)
-		//TODO: log of given vs. returned rule and result
-		if err != nil {
-			rule, result, _ = s.DecideAnnouncement(s.RuleToVote, s.VotingResult)
-		}
-	} else {
-		rule, result, _ = s.DecideAnnouncement(s.RuleToVote, s.VotingResult)
-	}
-
-	//Reset
-	s.RuleToVote = ""
-	s.VotingResult = false
-
-	if rule != "" {
-		//Deduct action cost
-		s.budget -= 10
-
-		//Perform announcement
-		//broadcastToAllIslands(s.Id, generateVotingResultMessage(rule, result))
-		return s.updateRules(rule, result)
-	}
-
-	return nil
-}
+//
+//func (s *baseSpeaker) announceVotingResultTest() error {
```

Comment:
what?

Reply:
Test is a little redundant now and will be impacted heavily by post-mvp changes so we commented it out for now instead of fixing